_posts/2022-06-30-trino-summit-call-for-speakers.md

```diff
@@ -42,4 +42,7 @@ lineup with talks from the broader Trino community. If you're interested, you
 can check out the speaker registration page for more information.
 
 And of course, we're looking forward to seeing you there, whether in-person or
-online!
```

Comment:
Let's also remove the "Sign up to speak" button on lines 20-22.

Reply:
Done